documentation/tutorials/customizing-tokens-via-cli.md

```diff
@@ -0,0 +1,95 @@
+# Customizing tokens via CLI
+
+In this tutorial, we'll dig into more details and go beyond what the dApp allows.
+We'll customize the parameters of synthetic tokens via the command line by creating a BTC token
```

Comment:
Do you think we should refer to it as a BTC/USD token or do you think that would make the quanto/compo thing more confusing?

Reply:
Either way, what do you prefer?

Reply:
Done.